Sources/MessagingHelpers/VaporTransport.swift

```diff
@@ -129,10 +144,28 @@ extension URLSession {
         let data = try BSONEncoder().encode(body).makeData()
         
         if data.count > maxBodySize {
+#if canImport(FoundationNetworking)
+            guard let res = URLResponse(coder: NSCoder()) else { throw  VaporTransportError.urlResponseNil }
```

Comment:
I'm not a fan of trying to use NSCoder.

Reply:
I think we can initialize URLResponse differently, but I don't think that in FoundationNetworking it comes with an empty initializer, like in Foundation. https://github.com/apple/swift-corelibs-foundation/blob/eec4b26deee34edb7664ddd9c1222492a399d122/Sources/FoundationNetworking/URLResponse.swift